Bottom: ac5ec6cd86337283abb0addb31065cd751834f9b
Top:    3d29ffacfb74b5d90fa9d14fbde5aac100ff3c54
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-12-28 11:51:46 +0800

Refresh of tun-rx-batching

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index c2af7a3..030286c 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -1331,7 +1331,7 @@ static ssize_t tun_get_user(struct tun_struct *tun, struct tun_file *tfile,
 	} else
 		err = tun_rx_batched(tfile, skb, more);
 #else
-		netif_rx_ni(skb);
+	netif_rx_ni(skb);
 #endif
 
 	stats = get_cpu_ptr(tun->pcpu_stats);
